bundles/framework/divmanazer/component/ColorPickerInput.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * @class Oskari.userinterface.component.ColorPickerInput
+ *
+ * Color picker component with default color palette
+ */
+Oskari.clazz.define('Oskari.userinterface.component.ColorPickerInput',
+
+    /**
+     * @method create called automatically on construction
+     * @param object options colorPicker options for customizing spectrum component
+     */
+    function (options) {
+        var me = this;
+        me._localization = Oskari.getLocalization("DivManazer");
```

Comment:
As the console says: "Oskari.getLocalization() will be removed in future release. Use Oskari.getMsg() instead." Other than that, good for merging.